x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details_content.tsx

```diff
@@ -70,83 +74,98 @@ export const EndpointDetailsContent = memo(
     const detailsResults = useMemo(() => {
       return [
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.os', {
-            defaultMessage: 'OS',
-          }),
-          description: <EuiText>{details.host.os.full}</EuiText>,
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.os"
+                defaultMessage="OS"
+              />
+            </ColumnTitle>
+          ),
+          description: <EuiText size="xs">{details.host.os.full}</EuiText>,
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.agentStatus', {
-            defaultMessage: 'Agent Status',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.agentStatus"
+                defaultMessage="Agent Status"
+              />
+            </ColumnTitle>
+          ),
           description: <EndpointAgentStatus hostStatus={hostStatus} endpointMetadata={details} />,
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.lastSeen', {
-            defaultMessage: 'Last Seen',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.lastSeen"
+                defaultMessage="Last Seen"
+              />
+            </ColumnTitle>
+          ),
           description: (
-            <EuiText>
+            <EuiText size="xs">
               {' '}
```

Comment:
Not part of your changes, but just wondering if this was intentional or if perhaps it was just the IDE formatter that did it

Reply:
This is probably some previous PR mistake. I'll remove it.